config/webpack.dev.js

```diff
@@ -90,7 +96,7 @@ module.exports = {
       // in development "style" loader enables hot editing of CSS.
       {
         test: /\.css$/,
-        loader: 'style!css!postcss'
+        loader: ExtractTextPlugin.extract('style-loader', 'css-loader', 'postcss-loader')
```

Comment:
ExtractTextPlugin doesn't work well with HMR https://github.com/webpack/extract-text-webpack-plugin/issues/222 Better to use it only in production configuration